examples_tests/mujoco/test_td3.sh

```diff
@@ -1,12 +0,0 @@
-#!/bin/bash
-
-set -Ceu
-
-outdir=$(mktemp -d)
-
-gpu="$1"
-
```

Comment:
Shouldn't this file be deleted?

Reply:
This file was not removed but renamed https://github.com/chainer/chainerrl/pull/487/commits/cd7180b1bb175355a4b49a78652b32249b8f386d#diff-9784aa586d342815241085cc9b58991a
(The final diff shows misleading results because I fixed the content of files at same time)